src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ReferralController.kt

```diff
@@ -121,16 +122,16 @@ class ReferralController(
     @Nullable @RequestParam(name = "cancelled", required = false) cancelled: Boolean?,
     @Nullable @RequestParam(name = "unassigned", required = false) unassigned: Boolean?,
     @Nullable @RequestParam(name = "assignedTo", required = false) assignedToUserId: String?,
-  ): List<SentReferralSummaryDTO> {
+  ): List<SentReferralSummaryForDashboardDTO> {
```

Comment:
we can't really change the response types of these existing endpoints, since the new responses are not backwards compatible with the old responses.

Reply:
I really was worried about that too but the reason why I changed this DTO is because the UI code which was calling this endpoint got changed to the new endpoint and there is no one calling the old endpoint. All the UI related code like the dashboardPresenter got changed to cater the new endpoint. If we have to keep both the api's with different models, we have to maintain two different service methods(one for fetching the the model and other for the old one). That urged me to make this change but I am 100% ok to revert this change.